torch/_dynamo/variables/tensor.py

```diff
@@ -681,36 +684,59 @@ class TensorWithTFOverrideVariable(VariableTracker):
     def create(
```

Comment:
It would really be helpful to have a top-level comment describing the general strategy and the other possible regimes you could live in here. Basically the stuff we talked about over VC a few weeks ago.

Reply:
I'll write up a doc